spec/models/alchemy/page_spec.rb

```diff
@@ -1532,6 +1532,20 @@ module Alchemy
           it { is_expected.to be(false) }
         end
       end
+
+      context "template has no alchemy role defined" do
+        before do
+          allow(page).to receive(:definition).and_return({"editable_by" => []})
```

Comment:
I think it's better to return an empty definition here. Since now the key is still present. What about, when the key is not present, aka nil?